docs/CONFIG-VARS.md

```diff
@@ -255,10 +261,10 @@ V4_CFG_POSTGRES_SERVERS:
 | backrest_pvc_storage_size | Size of the internal pgBackrest PVCs | string | 128Gi | false | This value can be changed but not decreased after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
 | postgres_pvc_access_mode | Access mode for the PostgreSQL PVCs | string | ReadWriteOnce | false | Supported values: [`ReadWriteOnce`,`ReadWriteMany`]. This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
 | backrest_pvc_access_mode | Access mode for the pgBackrest PVCs | string | ReadWriteOnce | false | Supported values: [`ReadWriteOnce`,`ReadWriteMany`]. This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
-| postgres_storage_class | Storage class for the PostgreSQL PVCs | string | sas | false |This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
-| backrest_storage_class | Storage class for the pgBackrest PVCs | string | sas | false |This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases.| viya |
+| postgres_storage_class | Storage class for the PostgreSQL PVCs | string | pg-storage | false |This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases. The default value is only used if `V4_CFG_MANAGE_STORAGE` is set to `true`. If `V4_CFG_MANAGE_STORAGE` is set to false, set the value to your pre-existing StorageClass name. | viya |
```

Comment:
I would get clarification on the wording here. The last sentence seems a bit off. Have Susan or someone help with the wording.

Reply:
@supear, For the last sentence, I am trying to convey that if V4_CFG_MANAGE_STORAGE is set to false, viya4-monitoring will not create the default pg-storage storage class so the name of a pre-existing storage class must be provided. Can you suggest alternate wording to convey that more clearly than above?
Maybe I should also change the 2nd to last sentence to say: "The default pg-storage storage class is only created when `V4_CFG_MANAGE_STORAGE` is set to `true`.

Reply:
@thpang here's that section with Susan's update, how's that sound now?
```suggestion
| postgres_storage_class | Storage class for the PostgreSQL PVCs | string | pg-storage | false |This value cannot be changed after the initial deployment. Supported for cadence versions 2022.10 and later. Only for internal databases. When `V4_CFG_MANAGE_STORAGE` is set to `true`, a storage class named `pg-storage` is created and is configured as the default storageClass. If `V4_CFG_MANAGE_STORAGE` is set to false, specify the name of an existing storage class for the value. | viya |
```